Test-Source/Invoke-DockerBuild.Tests.ps1

```diff
@@ -86,15 +76,24 @@ Describe 'Build docker images' {
         }
 
         It 'can consume arguments from pipeline' {
-            Mock -CommandName "Invoke-Command" $returnExitCodeZero -Verifiable -ModuleName $Global:ModuleName
             & $pipedInput | Invoke-DockerBuild
         }
 
         It 'returns the expected pscustomobject' {
-            Mock -CommandName "Invoke-Command" $returnExitCodeZero -Verifiable -ModuleName $Global:ModuleName
             $result = & $pipedInput | Invoke-DockerBuild
             $result.Dockerfile | Should -Not -BeNullOrEmpty
             $result.ImageName | Should -Not -BeNullOrEmpty
         }
     }
+
+    Context 'Passthru execution' {
+
+        It 'Captures the output of the command invoked' {
+            $tempFile = New-TemporaryFile
+            Invoke-DockerBuild -ImageName "leeandrasmus" -Dockerfile $dockerFile -Passthru 6> $tempFile
```

Comment:
`6>` ps has 6th stream?

Reply:
Yes, that is where Write-Information goes since PS 5.0